Source/Core/Core/Movie.h

```diff
@@ -102,14 +102,15 @@ struct DTMHeader
   bool bSyncGPU;
   bool bNetPlay;
   bool bPAL60;
+  bool bReducePollingRate;
```

Comment:
You can't shift around the other fields of this struct. Please only add new fields in places that were reserved before (like right below `language`, if you decrease the size of `reserved` instead of `reserved2`).

Reply:
Sure, though I'm a bit confused about why this actually matters, as DTMs are not compatible across Dolphin revisions.

Reply:
Ah, nevermind, I see the comment above now.